### ISSUE TITLE
Could you add support for dynamic language change?

### ISSUE THREAD

**Issue by JosefJezek** [firebase/firebaseui-web#789]
Could you add support for dynamic language change?

ES module per language has about 200KB. :-(

esm__cs.js  205KB
esm__en.js 199KB
...

**Comment by bojeil-google** [comment]
Hey @JosefJezek, our localization mechanism generates a new binary per language. This works by importing a specific binary based on a specific language (you shouldn't import all languages at the same time). You should be able to build your own language change mechanism based on that, for example, you could have the language in the url path and based on the path always import the specific localized binary.
Even if we support dynamic languages change, it is unlikely we will bundle all the translations into one binary (there are a lot of languages that we support). A server round trip will still likely be required.

**Comment by JosefJezek** [comment]
Could you create a base module + small modules per language?

**Comment by landsman** [comment]
> Could you create a base module + small modules per language?

Exactly, this can be dynamically loaded after the user selects the language.
Good inspiration: https://lingui.js.org/guides/dynamic-loading-catalogs.html#final-i18n-loader-helper

**Comment by landsman** [comment]
Okay, now I realized how this is made 🥵 : 
https://github.com/firebase/firebaseui-web-react/issues/21#issuecomment-887456061

**Comment by landsman** [comment]
Related with #662.

**Comment by dackers86** [comment]
Thank you for taking the time to submit this issue.

A new release is planned shortly, which should resolve this request. 

In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.

**Comment by landsman** [comment]
> Thank you for taking the time to submit this issue.
> 
> A new release is planned shortly, which should resolve this request.
> 
> In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.

What release? There is even not any beta release to let us test it first: https://github.com/firebase/firebaseui-web/releases

It's a good manner to close issues when it's actually done, not before.
